diff --git a/lib/ZnapZend.pm b/lib/ZnapZend.pm
index df5d310..db0976a 100644
--- a/lib/ZnapZend.pm
+++ b/lib/ZnapZend.pm
@@ -2,7 +2,7 @@ package ZnapZend;
 
 use Mojo::Base -base;
 use Mojo::IOLoop::ForkCall;
-use Mojo::Util qw(slurp);
+use Mojo::File qw(path);
 use Mojo::Log;
 use ZnapZend::Config;
 use ZnapZend::ZFS;
@@ -638,7 +638,7 @@ my $daemonize = sub {
     my $pidFile = $self->pidfile || $self->defaultPidFile;
 
     if (-f $pidFile){
-        chomp(my $pid = slurp $pidFile);
+        chomp(my $pid = path($pidFile)->slurp);
         #pid is not empty and is numeric
         if ($pid && ($pid = int($pid)) && kill 0, $pid){
             die "I Quit! Another copy of znapzend ($pid) seems to be running. See $pidFile\n";
